src/redux/actionTypes/projects.js

```diff
@@ -50,6 +50,11 @@ const PROJECTS_METADATA_UPDATE = `${PROJECTS}/metadataUpdate`;
  */
 const PROJECTS_METADATA_DELETE = `${PROJECTS}/metadataDelete`;
 
+/**
+ * Load the projects
+ */
+const PROJECTS_LOAD = `${PROJECTS}/load`;
+
```

Comment:
you should also have an action type called `PROJECTS_LOADING` which will be equal to `${PROJECTS}/loading` and will be dispatched just before you call the API endpoint to load projects (I will add a comment to the correct place).